addons/web/static/src/core/errors/error_handlers.js

```diff
@@ -148,6 +150,18 @@ const defaultDialogs = new Map([
  * @returns {boolean}
  */
 function defaultHandler(env, error) {
+    if (
+        error.constructor === UncaughtCorsError &&
+        isIOS() &&
+        session.is_frontend &&
+        odoo.debug !== "assets"
+    ) {
+        // In Safari 16.4+ (as of Jun 14th 2023), an error occurs
+        // when going back and forward through the browser when the
+        // cache is enabled. A feedback has been reported but in the
+        // meantime, hide any script error in these versions.
+        return;
+    }
```

Comment:
Wouldn't it be better there? https://github.com/odoo/odoo/blame/dd0ccfd5cdf71da23878bfe655611e7a7630ca7a/addons/web/static/src/core/errors/error_service.js#L87
It would shortcircuit (`return;`) the error catching and would not even fire a `UncaughtCorsError` internally.
It's also where we already shortcircuit some error we don't want to really listen, as can be seen a few lines above (`errorsToIgnore`)